### PR TITLE
Responsive app menu

### PR DESCRIPTION
**Pull request by juliusknorr**
ezgif-2-4a1f35fd11

As discussed at the hackweek with @jancborchardt 

ToDo:
- [x] Fix loading icon
- [x] Enabling/disabling apps doesn't show up nicely

### CODE REVIEW

**Comment by icewind1991** [comment]
This manages to break the file list for me, it doesn't show any files in the list

file list

**Review by jancborchardt — DISMISSED** [review]
Can confirm @icewind1991's issue. It also results in a double scrollbar on the right in Firefox, and there are too few items shown by default (should be 8 icons).

(Screenshot is after scrolling, so the files are visible – shows double scrollbar and too few icons.)
capture du 2017-04-24 18-47-48

cc @juliushaertl

**Comment by juliushaertl** [comment]
Ready for review @nextcloud/designers

**Comment by juliushaertl** [comment]
@icewind1991 Thanks for having a look. Should be fixed now, there was some leftover css rule from development. :see_no_evil:

**Review by MorrisJobke — APPROVED** [review]
Tested and works (even in Edge ;)) 👍

File: core/css/header.scss [hunk 1]
```diff
@@ -300,20 +294,21 @@ nav {
...
+.app-loading {
+	.icon-loading-small-dark {
+		display: inline !important;
+		position: absolute;
+		left: 12px;
+		width: 16px;
+		height: 16px;
+	}
+	.app-icon {
+		-ms-filter: 'progid:DXImageTransform.Microsoft.Alpha(Opacity=0)';
```
Comment (skjnldsv) [inline core/css/header.scss]: Do we still need those ugly/strange opacity filters for ie?

**Review by skjnldsv — APPROVED** [review]
Incredible work! :)

**Review by jancborchardt — APPROVED** [review]
Works great now! :+1:

**Comment by jancborchardt** [comment]
Awesome work @juliushaertl! This is so cool :)

@MorrisJobke @LukasReschke @rullzer what do we do about the codecov patch diff?

**Comment by LukasReschke** [comment]
> @MorrisJobke @LukasReschke @rullzer what do we do about the codecov patch diff?

Ideally in a world where everything is awesome: Add tests for this 😉 

/me hides 😉

**Comment by LukasReschke** [comment]
… as that isn't the case => merged 😉

**Pull request merged**